package.json

```diff
@@ -7,30 +7,35 @@
     "postbuild": "next export",
     "start": "next start",
     "lint": "next lint",
-    "test": "jest --passWithNoTests"
+    "test": "jest --passWithNoTests",
+    "theme": "chakra-cli tokens /theme.tsx",
+    "theme:watch": "chakra-cli tokens /theme.tsx --watch",
+    "format": "prettier --write . --ignore-path .gitignore ",
+    "format:check": "prettier --check . --ignore-path .gitignore"
   },
   "dependencies": {
-    "@bjerk/brand": "^1.0.1",
-    "@emotion/react": "^11.7.1",
-    "@emotion/styled": "^11.6.0",
+    "@chakra-ui/icons": "^2.0.13",
+    "@chakra-ui/react": "^2.4.3",
+    "@chakra-ui/theme-tools": "^2.0.14",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
     "@mdx-js/react": "^1.6.22",
-    "@theme-ui/color": "^0.13.1",
-    "@theme-ui/prism": "^0.13.1",
-    "next": "12.1.0",
-    "next-theme-ui": "^1.2.0",
-    "react": "17.0.2",
+    "framer-motion": "^7.8.0",
+    "next": "13.0.6",
+    "react": "18.2.0",
     "react-calendly": "^2.2.3",
-    "react-dom": "17.0.2",
-    "react-intl": "^5.24.4",
-    "theme-ui": "^0.13.1"
+    "react-dom": "18.2.0",
+    "react-intl": "^5.24.4"
   },
   "prettier": "@cobraz/prettier",
   "devDependencies": {
-    "@bjerk/eslint-config": "^1.0.0",
+    "@bjerk/eslint-config": "^3.0.1",
```

Comment:
This is out of scope for this PR. This (double) major version bump produces breaking changes!